projects/story-decorator/decorator.ts

```diff
@@ -40,9 +43,35 @@ export const swcThemeDecoratorWithConfig =
                 render(story(), decorator);
             });
         }
+
+        let hideNavStyles;
```

Comment:
This hides the decorator's toolbar when the globals are being used so that we don't have a duplicate toolbar in the view.